### PR TITLE
Added Filter for Greater than or equal to

### PR DESCRIPTION
**Pull request by viveksoundrapandi**
Added filter and associated rspec

### CODE REVIEW

File: lib/wongi-engine/dsl.rb [hunk 1]
```diff
@@ -84,6 +84,9 @@ module Wongi::Engine::DSL
     clause :greater
     accept Wongi::Engine::GreaterThanTest
 
+    clause :greaterthanorequal
```
Comment (ulfurinn) [inline lib/wongi-engine/dsl.rb]: a `gte` alias would be good for readability

**Comment by ulfurinn** [comment]
perhaps add support for `<=` as well to keep things symmetric?

**Comment by viveksoundrapandy** [comment]
@ulfurinn with pleasure

File: lib/wongi-engine/dsl.rb (hunk 1 above)
Comment (viveksoundrapandy) [reply lib/wongi-engine/dsl.rb]: agreed

Comment (viveksoundrapandy) [reply lib/wongi-engine/dsl.rb]: done

**Comment by viveksoundrapandi** [comment]
@ulfurinn any updates?

**Pull request merged**